src/EnergyPlus/PackagedTerminalHeatPump.cc

```diff
@@ -2980,16 +2979,19 @@ namespace PackagedTerminalHeatPump {
 			MySizeFlag( PTUnitNum ) = false;
 		}
 
+		RhoAir = StdRhoAir;
+		PTUnit( PTUnitNum ).MaxCoolAirMassFlow = RhoAir * PTUnit( PTUnitNum ).MaxCoolAirVolFlow;
+		PTUnit( PTUnitNum ).MaxHeatAirMassFlow = RhoAir * PTUnit( PTUnitNum ).MaxHeatAirVolFlow;
+
```

Comment:
OK, now I see why this if needed. The problem I see is I only want to execute these lines once. So I am moving this all up into the sizing IF above. It also makes me wonder why we initialize Mdot again on next begin environment. We've always done that, but I don't know why.